README.md

```diff
@@ -1,11 +1,10 @@
 <a name="readme-top"></a>
 
 <div align="center">
-<img src="./app/assets/images/logo_white.png" width="150px">
+<img src="./app/assets/images/logo_white.png" width="170px">
   <h1><b>Personal BudgetApp  | Ruby on Rails Capstone Project</b></h1>
 </div>
```

Comment:
- [ ] Due to insufficient files changed I will leave a comment here. KIndly fix the routing issue as the app starts okay but I cannot sign up a new user or login due to a routing error. Due to this I am unable to check the app functionality

![image](https://github.com/Ruthmy/budget-ap
![image](https://github.com/Ruthmy/budget-app/assets/81932478/580c6730-7418-4710-91fe-33e654d50229)
p/assets/81932478/5575cb03-aaf4-4279-8913-73241721108f)

### Kindly ensure you use the given font family as per requirement. I can see you have used a different one, while it looks good, we need to follow the given design in terms of the font family as well.